init-tools.cmd

```diff
@@ -1,6 +1,9 @@
 @echo off
 setlocal
 
+REM Workaround https://github.com/dotnet/coreclr/issues/2153
+set ComPlus_ReadyToRun=0
```

Comment:
Yes I was referring to building an individual csproj file (not a .builds file as all traversal projects disable restore by default). If it is scoped to just csc then it should be good enough to put this only in init-tools as that is the only place we do dotnet publish.